client/src/components/Chat.js

```diff
@@ -0,0 +1,104 @@
+import React, { useState, useEffect, useRef } from "react";
+import { useParams } from 'react-router-dom';
+import io from "socket.io-client";
+import { makeStyles, Box, Paper, TextField } from '@material-ui/core';
+import ChatWindow from '../components/ChatWindow';
+import { useUserState } from '../ContextProvider/user';
+
+
+const useStyles = makeStyles({
+    card: {
+        display: 'grid',
+        gridTemplateColumns: '1fr',
+        margin: '30px',
+        minheight: '10vh',
+        maxWidth: '35vw',
+    },
+    button: {
+        marginTop: '10px',
+        marginLeft: '10px',
+        padding: '10px',
+        background: 'transparent',
+        borderRadius: '5px',
+    },
+    window: {
+        height: '60vh',
+        maxHeight: '60vh',
+        maxWidth: '20vw',
+    },
+    message: {
+        width: '75%',
+    },
+});
+
+const Chat = () => {
+    const [message, setMessage] = useState('');
+	const [ chat, setChat ] = useState([]);
+    const { user } = useUserState();
+    const { matchId } = useParams();
+	const socketRef = useRef();
+    const classes = useStyles();
```

Comment:
Can you run prettier and update the spacing here?

Reply:
used prettier to fix spacing